pkg/console/starter/starter.go

```diff
@@ -108,13 +111,37 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		oauthinformers.WithTweakListOptions(tweakListOptionsForOAuth),
 	)
 
-	recorder := ctx.EventRecorder
+	resourceSyncerInformers := v1helpers.NewKubeInformersForNamespaces(
+		kubeClient,
+		api.TargetNamespace,
+		api.OpenshiftConfigManagedNamespace,
+		api.OpenShiftConsoleOperatorNamespace,
+	)
 
 	operatorClient := &operatorclient.OperatorClient{
 		Informers: operatorConfigInformers,
 		Client:    operatorConfigClient.OperatorV1(),
 	}
 
+	resourceSyncer := resourcesynccontroller.NewResourceSyncController(
+		operatorClient,
+		resourceSyncerInformers,
+		v1helpers.CachedSecretGetter(kubeClient.CoreV1(), resourceSyncerInformers),
+		v1helpers.CachedConfigMapGetter(kubeClient.CoreV1(), resourceSyncerInformers),
+		ctx.EventRecorder,
+	)
+
+	// add syncing for the managed ConfigMap
```

Comment:
NIT: I may change the comment from `what` to `why`:
`add syncing for the managed ConfigMap` 
vs 
`We need to sync the console-config configmap from openshift-config-managed because it may contain necessary branding configuration.`